pymc3/tests/test_smc.py

```diff
@@ -270,28 +468,38 @@ def test_automatic_use_of_sort(self):
             )
         assert s_k.distribution.sum_stat is pm.distributions.simulator.identity
 
-    def test_repr_latex(self):
-        expected = "$\\text{s} \\sim  \\text{Simulator}(\\text{normal_sim}(a, b), \\text{gaussian}, \\text{sort})$"
-        assert expected == self.s._repr_latex_()
-        assert self.s._repr_latex_() == self.s.__latex__()
-        assert self.SMABC_test.model._repr_latex_() == self.SMABC_test.model.__latex__()
-
     def test_name_is_string_type(self):
         with self.SMABC_potential:
             assert not self.SMABC_potential.name
-            trace = pm.sample_smc(draws=10, kernel="ABC")
+            trace = pm.sample_smc(draws=10, cores=1, return_inferencedata=False)
             assert isinstance(trace._straces[0].name, str)
 
     def test_named_models_are_unsupported(self):
-        def normal_sim(a, b):
-            return np.random.normal(a, b, 1000)
-
         with pm.Model(name="NamedModel"):
             a = pm.Normal("a", mu=0, sigma=1)
             b = pm.HalfNormal("b", sigma=1)
-            c = pm.Potential("c", pm.math.switch(a > 0, 0, -np.inf))
-            s = pm.Simulator(
-                "s", normal_sim, params=(a, b), sum_stat="sort", epsilon=1, observed=self.data
-            )
+            s = pm.Simulator("s", NormalSimRV1(), a, b, observed=self.data)
+
+            # TODO: Why is this?
```

Comment:
The exception message links to my answer here: https://github.com/pymc-devs/pymc3/pull/4365#issuecomment-761221146

If you read it you can probably tell if that's outdated with your latest changes or not.